src/WorkItems/FilingClientFactory.cs

```diff
@@ -26,7 +26,7 @@ public static FilingClient Create(Uri filingHostUri)
                 if (match.Success)
                 {
                     filingClient = isGitHub ? (FilingClient)new GitHubFilingClient() : new AzureDevOpsFilingClient();
-                    filingClient.CurrentProvider = isGitHub ? FilingClient.SourceControlProvider.Github : FilingClient.SourceControlProvider.Github;
+                    filingClient.CurrentProvider = isGitHub ? FilingClient.SourceControlProvider.Github : FilingClient.SourceControlProvider.AzureDevOps;
```

Comment:
>CurrentProvider [](start = 33, length = 15)

Do me a favor, change this property name to 'Provider' since I don't think we allow swapping providers in an out w/a common instance. And so 'Current' is a bit of a misqualification.

Reply:
Renamed to Provider.

---
In reply to: [398645608](https://github.com/microsoft/sarif-sdk/pull/1830#discussion_r398645608) [](ancestors = 398645608)